src/components/TransactionsTab/TransactionsTab.tsx

```diff
@@ -150,6 +170,57 @@ const TransactionsTab = ({
 
   const balance = balances.values[nativeCurrencyAddress] || "0";
 
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent) => {
```

Comment:
Use blur, as I said in discord earlier:

`and then when handleSelectBlur in Dropdown is triggered you propogate that back to the parent and then you hide the Dropdown again`